packages/react/src/avatar/Avatar.css.ts

```diff
@@ -44,16 +44,16 @@ export const avatar = recipe({
         yellow: "yellow",
       } as const,
       (color) => ({
-        bg: `${color}.50`,
-        color: `${color}.500`,
+        bg: `${color}.500`,
+        color: "white",
       }),
     ),
     size: {
       xs: { fontSize: "xs", size: "xs" },
-      sm: { fontSize: "sm", size: "sm" },
+      sm: { fontSize: "xs", size: "sm" },
       md: { fontSize: "md", size: "md" },
-      lg: { fontSize: "lg", size: "lg" },
-      xl: { fontSize: "xl", size: "xl" },
+      lg: { fontSize: "2xl", size: "48" },
+      xl: { fontSize: "4xl", size: "80" },
```

Comment:
let's add a new `5xl` size token for 80 and use it here
```suggestion
      xl: { fontSize: "4xl", size: "5xl" },
```